Packs/CommonScripts/Scripts/Jmespath/jmespath.yml

```diff
@@ -0,0 +1,23 @@
+commonfields:
+  id: jmespath
+  version: -1
+name: jmespath
+script: ''
+type: python
+tags:
+- transformer
+- general
+comment: Performs jmespath search on input JSON.
+enabled: true
+args:
```

Comment:
you are missing the `value` here, but i would rename that to data, would be more appropriate IMHO